### PR TITLE
Upgrade to 0.19

### PR DESCRIPTION
**Pull request by w0rm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
-elm-stuff/
+elm-stuff
 documentation.json
 tests/Doc
 node_modules
+.DS_Store
```
Comment (BrianHicks) [inline .gitignore]: platform-specific ignores belong in a global config, I'd rather keep them out of this repo. :)

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+package-lock.json
```
Comment (BrianHicks) [inline .gitignore]: shouldn't we check in package-lock.json?

File: .travis.yml [hunk 3]
```diff
@@ -1,30 +1,4 @@
...
-      cd ..;
-    fi
-
-  - make deps
-
-script:
-  - $TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 make documentation.json
-  - $TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 make test
+node_js: node # latest stable
+script: make
```
Comment (BrianHicks) [inline .travis.yml:4]: do you not have to build documentation.json separately in 0.19? I'd like that to stay in CI if possible.

Also, AFAICT the sysconfcpus bug is not fixed in 0.19. Did you find differently?

**Comment by w0rm** [comment]
Tests definitely passed locally :(

File: src/Trend/Math.elm [hunk 4]
```diff
@@ -48,8 +45,8 @@ mean numbers =
 
 {-| Calculate the standard deviation for some values.
 
-    mean [ 1, 2, 3, 4, 5 ]
-        --> Ok 3
+    stddev [ 1, 2, 3, 4, 5 ]
```
Comment (BrianHicks) [inline src/Trend/Math.elm:48]: thanks for the catch here!

**Review by BrianHicks — CHANGES_REQUESTED** [review]
this is looking great, thank you for taking the time to upgrade it! Just a couple comments about meta-stuff. 😄

File: .gitignore (hunk 2 above)
Comment (w0rm) [reply .gitignore]: It's a huge file to publish, but I can check it in if you want!

File: .travis.yml (hunk 3 above)
Comment (w0rm) [reply .travis.yml:4]: 1) `make` runs `make all` that includes `documentation.json test`.
2) I removed sysconfcpus from my game and it compiled within a minute. It took a bit more than a minute for https://travis-ci.org/BrianHicks/elm-trend/builds/422174067 to complete, so I don't find this necessary, but I can restore it if you want.

File: .gitignore (hunk 1 above)
Comment (w0rm) [reply .gitignore]: Removed!

File: .travis.yml (hunk 3 above)
Comment (BrianHicks) [reply .travis.yml:4]: as long as it's under 5 I think we're good. Thanks 😄

File: .gitignore (hunk 2 above)
Comment (BrianHicks) [reply .gitignore]: I dunno! Is it the right thing to do in this case? I'm less familiar with how node works—I'm only using it as a delivery mechanism for Elm binaries in this project :joy:

**Comment by w0rm** [comment]
@BrianHicks I believe I addressed all your comments!

**Comment by BrianHicks** [comment]
yeah, I think you did! Do you think it'd be fine to just publish the new version on 0.19 since there are no substantial changes? I'm not sure how it will need to be tagged. Since you've done a couple of them, and guidance?

**Comment by BrianHicks** [comment]
(I'll merge when we have a solid plan there.)

**Review by BrianHicks — APPROVED** [review]

**Comment by w0rm** [comment]
Just merge, run `elm bump`, commit the change, then tag on github the same way as before, and publish with `elm publish`.

**Comment by BrianHicks** [comment]
Great, thank you! I'll merge and do that soon :)

**Pull request merged**